tutorials/src/bin/basic-tutorial-2.rs

```diff
@@ -29,16 +29,15 @@ fn main() {
 
     // Wait until error or EOS
     let bus = pipeline.get_bus().unwrap();
-    if let Some(msg) = bus.timed_pop(gst::CLOCK_TIME_NONE) {
+    while let Some(msg) = bus.timed_pop(gst::CLOCK_TIME_NONE) {
         use gst::MessageView;
         match msg.view() {
             MessageView::Error(err) => {
                 eprintln!("Error received from element {}: {}",
                          msg.get_src().get_path_string(), err.get_error());
                 eprintln!("Debugging information: {:?}", err.get_debug());
```

Comment:
On Error you also want to break. I'll fix that up and merge it, thanks!

Reply:
Oh, I completely missed that obvious part! 
Thank you for all your feedback, it's super helpful!

Reply:
I'm glad to hear that it was useful for you :) Thanks again for this PR, it's merged now